CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project uses a version scheme based on the Cadenza main version in the format x.x.y, where x.x is the Cadenza main version and y a functional change or bugfix.
 
 ## Unreleased
+### Fixed
+- Removed duplicate geometry section in `sandbox.html#editGeometry`
+
+### Changed
+- Examples are now displayed more compact
```

Comment:
- Examples in the sandbox now are displayed more compact

Reply:
Or:
> Improved the examples in the sandbox to avoid a horizontal scrollbar.